app/models/zotero_api/bibliography.rb

```diff
@@ -0,0 +1,21 @@
+module ZoteroApi
+  # A complete Zotero bibliography
+  class Bibliography < Array
+    # @return [Bibliography] all items sorted by Author-Date
+    def sort_by_author_date
+      self.class.new(sort_by(&:to_author_date))
+    end
+
+    # @param [String] `key` the bibliography item's `key` value
+    # @return [BibliographyItem] the matching item, or `nil`
+    def find(key)
+      each { |item| return item if item['key'] == key }
+      nil
+    end
+
+    # @return [String] HTML for the entire bibliography, sorted by Author-Date
+    def render
+      "<ul>#{sort_by_author_date.collect(&:to_html).join}</ul>"
```

Comment:
Nah, there's some way you get to say something like "when you call `render @bibliography`, go render this partial". I'll dig it up.